tests/test_basic.py

```diff
@@ -490,9 +490,8 @@ def test_invalid_task_to_compute(self):
                 "There’s so much to learn when you’re slinging"
                 "paint and pencil"
             )
-            s = msg.serialize()
             with self.assertRaises(exceptions.FieldError):
-                message.Message.deserialize(s, decrypt_func=None)
+                msg.serialize()
```

Comment:
Will slot value be verified during deserialization phase? On the receivers side?

Reply:
`deserialize_slot` works in a way that it will expect some message class, take the value and try to deserialize it into that class - I'd expect it will fail if the value is invalid, isn't that the case?